### PR TITLE
Merging in README additions.

### PR DESCRIPTION
**Pull request by DrLynch**
I made some additions to the bare README that reflect lessons learned in development and guidance about bcrypt.

### CODE REVIEW

**Comment by bradley-erickson** [comment]
@pmitros updated readme to clean up and make styling more consistent.

File: README.md [hunk 1]
```diff
@@ -97,8 +103,27 @@ All _Learning Observer_ development has been on Linux-based platforms
...
+platforms so you may need to install some version manually.
+
+#### Additional Notes
+
+At present the system also uses static content that is served from a repo.
+This allows us to actually select different sources for the static data.
+This can also point to the current copy if necessary and can be configured
+as part of the creds.yaml file to generate the repo or add it on startup.
+
+This is not ideal but it is a mechanism for embedding the alternatives.
```
Comment (pmitros) [inline README.md:124]: I do not agree. We're very deliberate to use formats based on git and Merkle trees with good reason. If you believe there is a better mechanism for version control, please let me know.

Comment (DrLynch) [reply README.md:124]: If you're bothered by that pov then we can delete it from the readme and discuss it offline.

Comment (DrLynch) [reply README.md]: Updated below.

**Pull request merged**